postgres/values.yaml

```diff
@@ -25,6 +25,16 @@ secret:
   # secretName:
   # postgresPassword:
 
+initData:
```

Comment:
Documentation about how use it missing here or in the README.

Reply:
Do I need to do more documentation than the values in the Expert table?